_data/gaming.yml

```
@@ -84,6 +84,14 @@ websites:
       - totp
     doc: https://www.epicgames.com/help/en-US/c74/c112/a3218
 
+  - name: ESL
+    url: https://account.eslgaming.com
```

The main url should not be a login page. Change it to this instead:
```suggestion
    url: https://eslgaming.com
```

---

Updated to www.eslgaming.com